jni4net.n/src/jni/MethodWrapper.cs

```diff
@@ -105,11 +105,8 @@ public static ModuleBuilder InitModuleBuilder()
         {
             if (moduleBuilder == null)
             {
-                assemblyBuilder =
-                    AppDomain.CurrentDomain.DefineDynamicAssembly(new AssemblyName("net.sf.jni4net.jni.dynammic"),
-                                                                  AssemblyBuilderAccess.RunAndSave); //TODO RUN only
-                moduleBuilder = assemblyBuilder.DefineDynamicModule("net.sf.jni4net.jni.dynammic",
-                                                                    "net.sf.jni4net.jni.dynammic.dll");
+                assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName("net.sf.jni4net.jni.dynammic"), AssemblyBuilderAccess.Run); 

```

Comment:
I forgot what this is good for ?

Reply:
It creates a dynamic assembly in order for the generator to register the functions in the runtime.

DefineDynamicAssembly has been removed from CurrentDomain in .net core

Reply:
Recent versions of .Net core are trying to get rid of reflection and dynamic code. The NativeAOT motivates this strongly.

Let's consider how to get rid of this and replace it with build time code gen.

Reply:
#2